.github/ISSUE_TEMPLATE/3-improvement-request.md

```diff
@@ -1,79 +1,62 @@
 ---
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
 name: Improvement request
-
-about: Suggest how to improve Traffic Control. Examples include usability, performance, tech debt, etc.
-
+about: Suggest how to improve an existing feature of Traffic Control (usability, performance, tech debt, etc.)
 labels: improvement
-
 ---
-
 <!--
 ************ STOP!! ************
 If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
 the Apache Traffic Control Security Team at security@trafficcontrol.apache.org and follow the
-guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+guidelines at https://apache.org/security regarding vulnerability disclosure.
 
 - For *SUPPORT QUESTIONS*, use the #traffic-control channel on the ASF slack (https://s.apache.org/slack-invite)
-or Traffic Control mailing lists (https://trafficcontrol.apache.org/mailing_lists).
-- Before submitting, please **SEARCH GITHUB** for a similar issue or PR.
+or the Traffic Control Users mailing list (send an email to users-subscribe@trafficcontrol.apache.org to subscribe).
+- Before submitting, please **SEARCH GITHUB** for a similar issue or PR
+    * https://github.com/apache/trafficcontrol/issues
+    * https://github.com/apache/trafficcontrol/pulls
 -->
 
 ## I'm submitting a ...
```

Comment:
should this just say "i'm submitting a improvement request related to:"

```
usability
performance
tech debt
other
```

Reply:
We can categorize it after the Issue is submitted, users shouldn't have to answer an additional question.

Changed *I'm submitting a ...* to a *Improvement request (usability, [performance](https://github.com/apache/trafficcontrol/issues?q=is:open+is:issue+label:performance+sort:updated-desc), [tech debt](https://github.com/apache/trafficcontrol/issues?q=is:open+is:issue+label:"tech+debt"+sort:updated-desc), etc.)* title in 6f12ee4525.